### ISSUE TITLE
Upgraded pip fails to install packages with enum34

### ISSUE THREAD

**Issue by ehsanmok** [aws/sagemaker-python-sdk#1469]
**Describe the bug**
Upgraded `pip` to the v20.1 fails to install GluonTS. Need to [remove `enum34`](https://github.com/iterative/dvc/issues/1995) from conda envs.

**To reproduce**

```
python -m pip install --upgrade -q pip
python -m pip install -q git+https://github.com/awslabs/gluon-ts@master 
```

**Screenshots or logs**
```
AttributeError: module 'enum' has no attribute 'IntFlag'
```

**System information**
A description of your system. Please provide:
- **SageMaker Python SDK version**: 1.56.1
- **Framework name (eg. PyTorch) or algorithm (eg. KMeans)**: MXNet but should affect all conda env
- **Framework version**: 1.6.0
- **Python version**: py36
- **Custom Docker image (Y/N)**: N

**Additional context**

Related to #1428

**Comment by nadiaya** [comment]
Just to clarify, you're referring to the mxnet_p36 environment in a SageMaker Notebook Instance?

**Comment by ehsanmok** [comment]
Correct!

**Comment by laurenyu** [comment]
looks like it's due to the installation of enum34 [here](https://github.com/aws/sagemaker-python-sdk/blob/master/setup.py#L82). Since this library still supports Python 2 (for now), we do need some kind of change that is compatible with Python 2 before blindly removing enum34.

as a workaround, are you able to manually uninstall enum34 before installing GluonTS?

**Comment by ehsanmok** [comment]
@laurenyu yes I can pip uninstall it in python3.6. Since SageMaker kernels are either p27 or p36 then it'd make sense to remove it as part of v2.0.0, py27 deprecation plan #1461

**Comment by laurenyu** [comment]
Glad you at least have a workaround for now.

re: v2.0.0 - yep, I was thinking the same thing 🙂. I've added it to the description of #1461, and will add this issue to the milestone/project board. If I get a chance, I'll still try to see if there's a Python 2-compatible solution, but from the quick Googling I did yesterday, pretty much everything seemed to point to enum34 🤷‍♀️

**Comment by laurenyu** [comment]
addressed in #1609 and slated for the v2.0.0.rc1 release

**Comment by laurenyu** [comment]
released in [v2.0.0.rc1](https://github.com/aws/sagemaker-python-sdk/releases/tag/v2.0.0.rc1)
